src/v2/view-builder/utils/Constants.js

```diff
@@ -11,4 +11,10 @@ export const OV_UV_ENABLE_BIOMETRICS_FASTPASS_DESKTOP
     = 'oie.authenticator.oktaverify.method.fastpass.verify.enable.biometrics.desktop';
 export const OV_UV_ENABLE_BIOMETRICS_FASTPASS_MOBILE 
     = 'oie.authenticator.oktaverify.method.fastpass.verify.enable.biometrics.mobile';
-
+export const REQUEST_PARAM_LOOPBACK_CANCEL_TRIGGER = 'triggeredByUser';
```

Comment:
Can this one be removed, because of AUTHENTICATION_CANCEL_REASONS.USER_CANCELED ?